script/Arcueid/Day 2/02_00_ARC02_2.txt

```diff
@@ -11,17 +11,19 @@ It's after school now
 [sha:1b94b3652a50b698382fcc487c23e1ed3cfadb12]{
 -- Page 10005, Offset 2988.
 -- 今日は貧血による目眩もなく、俺個人の話で言えばいたって平穏に過ごせた一日だった。
-I didn't feel any anemic dizziness today, and at least for me, the rest of the afternoon went by peacefully.
+I didn't suffer from any anemic episodes today,
+ and remainder of the afternoon passed peacefully, at least as far as I was concerned.
 }
 [sha:5459b733d1a07c982f37a3691cd5d8c4012c9809]{
 -- Page 10005, Offset 2989.
 -- しかし、教室では未曾有のウイルスが蔓延し、ついさっきまで不穏な空気に包まれていた。
-However, an unprecedented virus has infected the classroom, and until a short while ago, the air was filled with unrest.
+However, a novel virus has spread like wildfire through the classroom,
+ and up until a moment ago the atmosphere in here was heavy with unrest.
 }
 [sha:0b19246c1613d327ab52f9b20f24ad5477ed3e03]{
 -- Page 148, Offset 2990.
 -- 女子たちは今回の男子の騒ぎを『ノエル熱』と呼称した。
-The girls referred to this disturbance among the boys in the class as 'Noel Fever'.
+The girls dubbed this new affliction the boys were suffering from 'Noel Fever'.
```

Comment:
> suffering from as 'Noel

Reply:
I don't think you need it, but it sounds wrong to me without

Reply:
Hmm, It sounds pretty off to me with it there. Looking at other usages seems to suggest `as` is uncommon.
https://sentence.yourdictionary.com/dubbed

Reply:
🤷 don't really care that much either way